tests/frameworks/gtest/include/gtest-api.h

```diff
@@ -164,200 +167,200 @@ class ObjCTest : public ::testing::Test {
 // DOES NOT include file and line information in the log output
 #define LOG_INFO(format, ...) \
     std::cout << testing::Message() <<  \
-        "[  INFO ] " << GTestLogPrivate::Format(format, __VA_ARGS__).data() << std::endl
+        "[  INFO ] " << GTestLogPrivate::Format(format, ##__VA_ARGS__).data() << std::endl
```

Comment:
This isn't reformatting, I changed it from __VA_ARGS__ (ms-only) to ##__VA_ARGS__

---
In reply to: [72312790](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72312790) [](ancestors = 72312790)